packages/common/src/store/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/export */
```

Comment:
this seems like a standard export for us. Do we want to update the rule?

Reply:
so weirdly enough i could not for the life of me figure out why the errors were happening. it was saying stuff like duplicate import of `ADD` `DELETE` which are the action types, but then when i went to the index for that store slice, it did `export * as blah`, so i don't know how those actions were also being reexported....